htdocs/bom/bom_card.php

```diff
@@ -170,6 +170,13 @@
 		$disable_stock_change = GETPOST('disable_stock_change', 'int');
 		$efficiency = price2num(GETPOST('efficiency', 'alpha'));
 		$fk_unit = GETPOST('fk_unit', 'alphanohtml');
+
+		if (!empty($idprod) && $conf->workstation->enabled) {
```

Comment:
From v17, try to use isModEnabled('workstation') instead of conf->global ...

Reply:
Oh yes, sorry.. thank you !